ledger/participant-integration-api/src/main/scala/platform/index/LedgerBackedIndexService.scala

```diff
@@ -114,33 +114,26 @@ private[platform] final class LedgerBackedIndexService(
           verbose = verbose,
         )
         .map(_._2)
-    })
+    }
 
-  // Returns a function that memoizes the current end
-  // Can be used directly or shared throughout a request processing
-  private def convertOffset(implicit
+  private def convertOffset(offset: LedgerOffset)(implicit
```

Comment:
```suggestion
  private def convertOffset(offset: domain.LedgerOffset)(implicit
```

Reply:
Why do you want to prefix it?

Reply:
It seems to be prefixed elsewhere and imo it clarifies what is being converted.

Reply:
I think this is to differentiate it from the `LedgerOffset` protocol buffers message. IMO this is the wrong way around; we should prefix or alias the protobuf, not the domain type.

I'll add the prefix but I'll revisit this at some point to standardize it.